iaso/api/mobile/org_units.py

```diff
@@ -3,14 +3,22 @@
 from django.contrib.gis.geos import Point
 from django.core.cache import cache
 from rest_framework import viewsets, permissions
+from rest_framework.pagination import PageNumberPagination
 from rest_framework.response import Response
 
 from hat.api.export_utils import timestamp_to_utc_datetime
 from iaso.api.common import get_timestamp
 from iaso.api.common import safe_api_import
+from iaso.api.query_params import APP_ID, PAGE_SIZE, PAGE
 from iaso.models import OrgUnit, Project
 
 
+class MobileOrgUnitsSetPagination(PageNumberPagination):
+    page_size_query_param = PAGE_SIZE
+    page_query_param = PAGE
+    page_size = None  # None to disable pagination by default.
+
```

Comment:
Everywhere else, the paginations parameter is limit instead of page_size, I think. It would be better to stick with that

Reply:
That's a mistake. We should stick to limit. Do you use that parameter already on entities?

Reply:
Actually, the code on mobile passes `limit` :sweat_smile: , I'll change it in both places

Reply:
👍